packages/browser-wallet/src/popup/popupX/shared/Form/TokenAmount/TokenAmount.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import { atomFamily, selectAtom, useAtomValue } from 'jotai/utils';
+import { AccountAddress, AccountInfo, ContractAddress, CIS2 } from '@concordium/web-sdk';
+import { atom } from 'jotai';
+
+import { useSelectedAccountInfo } from '@popup/shared/AccountInfoListenerContext/AccountInfoListenerContext';
+import { contractBalancesFamily } from '@popup/store/token';
+import { ensureDefined } from '@shared/utils/basic-helpers';
```

Comment:
I can't figure out what the strategy is for all the things in `@popup/shared`. Do we move everything we use to `@popup/popupX/shared`? I reckon it would be much simpler to just keep everything in `@popup/shared` in its current place and run some tool to identify dead code when we're all done.

@Ivan-Mahda @limemloh what do you think?

Reply:
Moving things over could also be a source of conflicts, so my guess is also that it would be simpler to expand the current one

Reply:
Good point